lib/node_modules/@stdlib/stats/base/dists/bernoulli/pmf/docs/repl.txt

```diff
@@ -1,7 +1,7 @@
 
 {{alias}}( x, p )
     Evaluates the probability mass function (PMF) for a Bernoulli distribution
-    with success probability `p` at a value `x`.
+    with success probability `0 <= p <= 1`.
```

Comment:
This change is not needed. Below we already specify the range of valid values for `p`.